master/internal/db/postgres_tasks.go

```diff
@@ -249,6 +249,19 @@ func (db *PgDB) UpdateAllocationProxyAddress(a model.Allocation) error {
 	return err
 }
 
+// AddAllocationAcceleratorData stores acceleration data for an allocation.
+func (db *PgDB) AddAllocationAcceleratorData(containerID string, a model.Allocation,
+	nodeName string, acceleratorType string, acceleratorUuids []string,
+) error {
+	_, err := db.sql.Exec(`
```

Comment:
might be able to just do ```db.Bun().NewInsert().Model(accelData)``` if we pass in the model

Reply:
Done!